src/main/kotlin/net/leanix/vsm/gitlab/broker/connector/adapter/graphql/GitlabGraphqlProvider.kt

```diff
@@ -71,6 +77,24 @@ class GitlabGraphqlProvider(private val gitLabOnPremProperties: GitLabOnPremProp
             ?.toRepository()
             ?: throw GraphqlException("No gitlab project found at path: $nameWithNamespace")
 
+    override fun getMergeRequestsForRepository(
+        repository: Repository,
+        periodInDaysInString: String
```

Comment:
I don't think we need the **String** on the parameter name here.

Reply:
done